### PR TITLE
KIALI-3068 Create FAQ section

### PR DESCRIPTION
**Pull request by josejulio**
- Adding General FAQ
 - Moving known issues to Graph FAQ

### CODE REVIEW

**Comment by josejulio** [comment]
While not everything is ready, early feedback is welcome: https://deploy-preview-145--kiali-io-draft.netlify.com/faq

File: content/faq/general/how-do-i-get-started.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: How do i get started using Kiali?
+weight: 10
+---
+:icons: font
+
+We recommend following our link:/documentation/getting-started/[Getting Started] section which provides
+information on the instalation and configuration of Kiali.
```
Comment (jshaughn) [inline content/faq/general/how-do-i-get-started.adoc]: I don't think we need this FAQ entry.

File: content/faq/general/kiali-does-not-work.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: Kiali does not work - What do i do?
+weight: 40
+---
+:icons: font
+
+If you are hitting a limitation, regardless if it's listed here or not, do not hesitate to use the
```
Comment (jshaughn) [inline content/faq/general/kiali-does-not-work.adoc]: limitation -> problem
regardless if it's -> whether it is

File: content/faq/general/what-is-kiali.adoc [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: What is Kiali?
+weight: 0
+---
+:icons: font
+
+Kiali is an observability and configuration console for the Istio service mesh.
+
+Kiali project provides answers to the questions:
+What microservices are part of my Istio service mesh? How are they connected? How are they performing?
```
Comment (jshaughn) [inline content/faq/general/what-is-kiali.adoc]: I don't think we need this entry, it just repeats what is on the landing page.

File: content/faq/general/what-does-kiali-mean.adoc [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+title: What does the word "Kiali" mean?
+weight: 50
+---
+:icons: font
+
+κιάλι (Kiali) is the greek word for "monocular" or spyglass.
```
Comment (jshaughn) [inline content/faq/general/what-does-kiali-mean.adoc]: I don't think we need this entry, it just repeats what is on the landing page.

File: content/faq/graph/index.adoc [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+draft: false
+type: "known-issues"
+weight: 1
+layout: faq
+---
+
+:icons: font
+:imagesdir: /images/faq/
+
+This section describes some known issues in Kiali that may not be fixed in the very short term, and lead to limitations in a restricted number of corner cases. Hopefully, you wouldn't even notice them.
```
Comment (jshaughn) [inline content/faq/graph/index.adoc]: restricted number of corner cases. Hopefully, you wouldn't even notice them. -->
restricted number of cases.  Hopefully, you won't even notice them.

File: content/faq/graph/index.adoc [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+If you are hitting a limitation, regardless if it's listed here or not, do not hesitate to use the https://github.com/kiali/kiali/issues[GitHub issue tracker]. You can simply _vote_ (using emojis) for any existing bug or feature request, this will help us to understand the most demanded enhancements, or you can create a new ticket. For existing JIRA tickets, use the _vote_ link or add a comment describing your expectation or scenario.
```
Comment (jshaughn) [inline content/faq/graph/index.adoc]: limitation -> problem
regardless if it's -> whether it is

File: content/faq/graph/internet-explorer.adoc [hunk 7]
```diff
@@ -0,0 +1,9 @@
+---
+title: Kiali supports Internet Explorer?
```
Comment (jshaughn) [inline content/faq/graph/internet-explorer.adoc]: Kiali supports Internet Explorer?  -->
 Does Kiali support Internet Explorer?

**Comment by josejulio** [comment]
> * I think I'd rather see FAQ under documentation in order to limit the top-level menu.

OK, I think the FAQ should be in top-level menu, but no strong opinion on why.

> * I don't like the all-caps for the questions, should just be normal sentence capitalization , I think.

I will remove the caps and post a before/after picture for reference

> * General comment: I hate our web site's giant fonts, huge title bar and massive spacing.  Content should be much more compact so that users can see more without scrolling so much.  It's like looking at slides, or a kid's book.

OK, I'm not going to touch the overall design because I think is being re-designed.


> * I'd like to see a graph FAQ entries for
>   
>   * "Why do I have missing edges?
>   * "Why can't I see traffic leaving the mesh"

Ok !

**Comment by josejulio** [comment]
Updated, please review

File: content/faq/graph/internet-explorer.adoc [hunk 8]
```diff
@@ -0,0 +1,9 @@
+---
+title: Does Kiali support Internet Explorer?
```
Comment (jshaughn) [inline content/faq/graph/internet-explorer.adoc]: Seems like this should be in "general", not "graph".

Comment (josejulio) [reply content/faq/graph/internet-explorer.adoc]: True, will update and merge

**Pull request merged**